pom.xml

```diff
@@ -91,7 +81,19 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <maven.compiler.release>8</maven.compiler.release>
     </properties>
+    <profiles>
```

Comment:
## Author's Note

This profile is used to disable Javadoc generation on certain SDKs due to incompatibility: https://stackoverflow.com/a/16743137/13343799